src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -65,61 +67,65 @@
         <!-- Contact Info -->
         <contact-info :direction="'col'"/>
 
-        <div class="my-6">
-          <p class="subtitle">Benefit Company Articles</p>
-          <p class="info-text">Before submitting your alteration notice you <span class="font-weight-bold">must change
-            your company's articles to include a set of Benefit Company Articles</span> OR draft new articles containing
-            a benefit provision.</p>
-        </div>
-
-        <!-- Help Section Toggle -->
-        <div class="info-text help-toggle pt-2" @click="helpToggle = !helpToggle">
-          <v-icon class="pr-2 mt-n2" color="primary">mdi-help-circle-outline</v-icon>
-          <span v-if="!helpToggle">Learn More</span>
-          <span v-else>Hide Learn More</span>
-        </div>
+        <template v-if="isBenefit">
+          <div class="my-6">
+            <p class="subtitle">Benefit Company Articles</p>
+            <p class="info-text">Before submitting your alteration notice you <span class="font-weight-bold">must
```

Comment:
Keep in mind that we might want to centralize and index these articles when there are more entity types in the future.
No action needed.

Reply:
And provisions, lol.

Reply:
For sure, when we have entities / documents etc to support, 100%